### DISCUSSION TITLE
Can i use this package outside of react-native?

### DISCUSSION THREAD

**Discussion by y-nk** [react-native-async-storage/async-storage#629] [General]
## You want to:  
Ask a question

## Details:
I'm feeling this package would be very good to have in the library i'm building. I'm aiming my library to be cross-platform, and i would like to avoid coding the localStorage part for the web ; but i cannot guarantee my library will be used in the context of react/react-native.

Would you recommend using this library as a "cross-platform" even outside of the use of react-native?

**Comment by krizzu** [comment]
Outside of React Native you mean web only? AsyncStorage works in React/ReactNative domain, and using it outside of it might produce more questions than answers.

**Comment by y-nk** [comment]
@Krizzu yes, i meant outside of react/react native.

aside from the hook, it looks to me like the code is mostly agnostic of frontend frameworks, so it could be targeting a wider range of web build, don't you think?
